_includes/v20.1/admin-ui/logical-bytes.md

```diff
@@ -0,0 +1 @@
+Logical bytes reflect the approximate number of bytes stored in the database. This value may deviate from the number of physical bytes on disk, due to factors such as compression and [write amplification](https://en.wikipedia.org/wiki/Write_amplification) affecting the latter.
```

Comment:
Suggest removing "affecting the latter".  It took me a minute to parse it, and I guess it means "write amplification affects compression", but it mostly adds confusion IMO since there is nothing you can do with that information.  The main message seems to be "these numbers will not match due to compression and write amplification", right?